src/Components/Cart/Cart.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+
+export default function Cart({ items, deleteCart }) {
+  let handleDelete = (id) => {
+    deleteCart(id);
+  };
+  var cartItems = items.map((value, index) => {
+    return (
+      <div className="cart-item" key={index}>
```

Comment:
using index as a key here is not very useful. Can you let me know why? Change key appropriately

Reply:
@Tush1999 - Please address this comment